packages/es-components-wtw-theme/index.js

```diff
@@ -481,7 +481,7 @@ const theme = {
   },
   inputStyles: {
     borderRadius: '2px',
-    defaultFormStyle: 'default',
+    defaultFormStyle: 'flat',
```

Comment:
I love this! I was just going to talk to some of the teams working on OneView about this. 🎉